.github/workflows/tests.yml

```diff
@@ -0,0 +1,404 @@
+              TEST_SCRIPT: tests.sh
+              OPEN_RUNTIMES_BUILD_COMMAND: ""
+              OPEN_RUNTIMES_START_COMMAND: "java -jar src/function/java-runtime-1.0.0.jar"
+            - runtimes: java-11.0
+              RUNTIME: java-11.0
+              TEST_CLASS: Base
+              ENTRYPOINT: Tests.java
+              IMAGE: openruntimes/java:v3-11.0
+              ARCH: linux/amd64,linux/arm64
+              TEST_SCRIPT: tests.sh
+              OPEN_RUNTIMES_BUILD_COMMAND: ""
+              OPEN_RUNTIMES_START_COMMAND: "java -jar src/function/java-runtime-1.0.0.jar"
+            - runtimes: java-8.0
+              RUNTIME: java-8.0
+              TEST_CLASS: Base
+              ENTRYPOINT: Tests.java
+              IMAGE: openruntimes/java:v3-8.0
+              ARCH: linux/amd64,linux/arm64
+              TEST_SCRIPT: tests.sh
+              OPEN_RUNTIMES_BUILD_COMMAND: ""
+              OPEN_RUNTIMES_START_COMMAND: "java -jar src/function/java-runtime-1.0.0.jar"
+            - runtimes: cpp-17
+              RUNTIME: cpp-17
+              TEST_CLASS: Base
+              ENTRYPOINT: tests.cc
+              IMAGE: openruntimes/cpp:v3-17
+              ARCH: linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64
+              TEST_SCRIPT: tests.sh
+              OPEN_RUNTIMES_BUILD_COMMAND: ""
+              OPEN_RUNTIMES_START_COMMAND: "src/function/cpp_runtime"
+            - runtimes: cpp-20
+              RUNTIME: cpp-20
+              TEST_CLASS: Base
+              ENTRYPOINT: tests.cc
+              IMAGE: openruntimes/cpp:v3-20
+              ARCH: linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64
+              TEST_SCRIPT: tests.sh
+              OPEN_RUNTIMES_BUILD_COMMAND: ""
+              OPEN_RUNTIMES_START_COMMAND: "src/function/cpp_runtime"
+            - runtimes: dotnet-7.0
+              RUNTIME: dotnet-7.0
+              TEST_CLASS: Base
+              ENTRYPOINT: Tests.cs
+              IMAGE: openruntimes/dotnet:v3-7.0
+              ARCH: linux/amd64,linux/arm64
+              TEST_SCRIPT: tests.sh
+              OPEN_RUNTIMES_BUILD_COMMAND: ""
+              OPEN_RUNTIMES_START_COMMAND: "dotnet src/function/DotNetRuntime.dll"
+            - runtimes: dotnet-6.0
+              RUNTIME: dotnet-6.0
+              TEST_CLASS: Base
+              ENTRYPOINT: Tests.cs
+              IMAGE: openruntimes/dotnet:v3-6.0
+              ARCH: linux/amd64,linux/arm64
+              TEST_SCRIPT: tests.sh
+              OPEN_RUNTIMES_BUILD_COMMAND: ""
+              OPEN_RUNTIMES_START_COMMAND: "dotnet src/function/DotNetRuntime.dll"
+    runs-on: ubuntu-20.04
+    name: ${{ matrix.runtimes }}
+    steps: 
+        - name: Clone Repository
+          uses: actions/checkout@v3.5.3
+        - name: Setup Docker Buildx
+          uses: docker/setup-buildx-action@v2.9.1
+        - name: Check Docker Buildx
+          run: docker buildx version
+        - name: Install PHP
+          uses: shivammathur/setup-php@2.25.4
```

Comment:
Can we look for a more official one ? If not, possibly the most well maintained one?

Reply:
@christyjacob4 there is no official setup-php action. 
This is was the best option I found.

If this is not needed how about I write a php and composer  installation command ?? rather than using an action.

Reply:
Let's use PHP inside docker instead if we can :+1:

Reply:
@abnegate That would require the same php and composer install command in every dockerfile of all the runtimes.(if i am correct)
How about I justt use a php and composer install command in a step rather than doing it for every dockerfile.

Reply:
@faisalill alright. We can keep using it then